changelog.txt

```diff
@@ -1,3 +1,286 @@
+Fix #11437: Flipped shorter rail vehicles disappear in windows (#11446)
+Fix #11413: Incorrect sorting by industry production (#11414)
+Fix #11407: Don't steal focus from dropdown menus (#11484)
+Fix #11402: Make string filter locale-aware (#11426)
+Fix #11329: Don't assert vehicle list length is non-zero when only asked to set string parameter (#11330)
+Fix #11315: Sort industries and cargoes by name in industry chain window (#11317)
+Fix #11307: Incorrect GroupStatistics after selling leading wagon (#11311)
+Fix #11261: Airport menu selectability after closing window on a class with no available airports (#11344)
+Fix #11230: Sort by button in group list window could be misaligned (#11231)
+Fix #11215: Assert in NewGRF parameters window (manual parameter mode) (#11217)
+Fix #11203: [Linux] Crash when editing CJK characters in edit box (#11204)
+Fix #11180: Aircraft crashes could point to the wrong tile (#11184)
+Fix #11164: Don't create duplicate town names when using 'Many random towns' in the scenario editor (#11165)
+Fix #11162: Second company colour was not consistently applied to articulated vehicles (#11163)
+Fix #11115: Focus the abandon game/exit game windows (#11125)
+Fix #11096: Increase priority of error and confirmation windows (#11104)
+Fix #11087: Disable base graphics/sound dropdown outside main menu (#11091)
+Fix #11054: Prevent translation of currency codes (#11061)
+Fix #11026: Use real engine name instead of default name for filtering (#11033)
+Fix #10982: No help text for gamelog command (#10984)
+Fix #10880: Crash in object window due to incorrect parameter order (#10881)
+Fix #10868: Crash when Script tries to load large savegame data (#11029)
+Fix #10811: Allow dragging vehicle in depot to any free row (#11508)
+Fix #10660: Sprite Font scale affected by viewport zoom level limits (#10668)
+Fix #10619: Crash loading linkgraph for older savegames (#10620)
+Fix #10600: 'Replace Vehicles' didn't show numbers >999 (#10680)
+Fix #10578: Allow to select any version of AI/GS from GUI (#10604)
+Fix #10522: Link graph tooltip vertical lines were not handled correctly (#10524)
+Fix #10511: Don't search for depot every tick if one cannot be found (#11548)
+Fix #10478: Clarify airport noise control setting texts (#11169)
+Fix #10452: Prevent long stalls during river generation (#11544)
+Fix #10430: Display chain window causing assert (#10431)
+Fix #10343: Don't extend town-disallowed roadtypes (#10347)
+Fix #10251: [MacOS] Screen looks blue-ish when using newer SDKs (#11207)
+Fix #10222: Adjust line drawing algorithm (#10491)
+Fix #10131: Actually cancel downloads when pressing cancel (#10485)
+Fix #10118: Cycle through current signal group, not just path signals (#11798)
+Fix #10439: [Script] Validate story page button colour, flags, cursor and vehicle type (#11892)
+Fix #10438: [Script] Validate story page element type for ScriptStoryPage::NewElement (#11888)
```

Comment:
Move these down, too?

Reply:
No, I sorted the fix numbers separately from the rest, so you can be amazed what is fixed :P